include/ringmesh/geomodel/geomodel_builder_topology.h

```diff
@@ -151,6 +141,21 @@ namespace RINGMesh
             const gmme_id& second_corner,
             const MeshType& mesh_type = "" );
 
+        void add_line_corner_boundary_relation(
+            index_t incident_line_id,
+            index_t boundary_corner_id );
+
+        void set_line_corner_boundary(
+            index_t incident_line_id,
+            index_t id,
+            index_t new_boundary_corner_id );
+
+        // TODO Could be protected (in public as long as the removal is not reworked) [BC]
+        // See comment in GeoModelBuilderRemoveBase< DIMENSION >::update_mesh_entity_boundaries [BC]
+        void set_mesh_entity_boundary(const gmme_id& gmme,
```

Comment:
I would suggest to put it protected and add a friendship to this method only. The goal is to remove this access from public section. Leaving it here is not good...